features/dynamic-bundle-discounts.md

```diff
@@ -130,6 +132,16 @@ Nosto Dynamic Bundle configuration involves two important keys, hash and secret
 
 Secret key, on the other hand, can be retrieved from "Dynamic Bundle Key" field in Nosto Admin Settings > Platform page. This value replaces GET\_FROM\_NOSTO placeholder in the Shopify line item script.
 
+## Scope
+In scope - Apply discounts when all the products in a bundle are added to cart. 
+Not in scope - Selecting only specific products from a bundle, for discounts.
+
+## Will not work If
```

Comment:
'will not work if' doesn't read well. I think it's better to

1. Add a sentence to the introduction to say that all products need to be added to the cart in order for the discount to be applied.
2. Add a warning to the Nosto Bundle Script section to warn that we can only provide support for the script as-is and we can't provide any support for scripts that have been modified.